packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -15,6 +16,11 @@ interface AutocompleteProps {
    */
   readonly value: Option | undefined;
 
+  /**
+   * Set Autocomplete value.
+   */
+  readonly debounceTime: number | undefined;
```

Comment:
I think we can not have this prop and start with a specific time for debounce until we get a specific use case that we actually need one. Unless you have a good reason as to why :D

also, the description is talking about a value

Reply:
whoops

Reply:
Do you think we can set is as optional for now or should we remove it at all? I was thinking that we might as well be able to customize this because if we have autocompletes with local data we might want it to not have a delay time. And for faster interactions, we might want to decrease the delay time. We can leave it as a default of 200ms maybe?